docs/docs/config/power.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Power Management Configuration
+sidebar_label: Power Management
+---
+
+See [Configuration Overview](index.md) for instructions on how to
+change these settings.
+
+## Idle/Sleep
+
+Configuration for entering low power modes when the keyboard is idle.
+
+In the idle state, peripherals such as displays and lighting are disabled, but the keyboard remains connected to Bluetooth so it can immediately respond when you press a key.
+
+In the deep sleep state, the keyboard additionally disconnects from Bluetooth. This state uses very little power, but it may take a few seconds to reconnect after waking.
```

Comment:
Is it worth noting what happens to external power in these two states?

Reply:
Maybe. What does happen in those states? Am I correct in understanding that it turns off external power in deep sleep but not idle?

Reply:
Yes, this is correct in the current state. In idle, external power stays on. In deep sleep, the external power is shut off.

Reply:
Documented what the (intended) behavior is.